packages/common/src/AppointmentUtils.ts

```diff
@@ -1,5 +1,6 @@
-import { DbAppointment } from "./types";
+import { BookingChange, DbAppointment } from "./types";
 import { AppointmentApiEntry } from "./functions-api";
+import * as admin from "firebase-admin";
```

Comment:
It only took 45m but I found the issue.

You can't import firebase-admin in common.

If you need `Timestamp` use `firebase.firestore.Timestamp` with `import firebase from "firebase";`